### ISSUE TITLE
https://print-nanny.com/dashboard/octoprint-devices/ page has too wide right side frame

### ISSUE THREAD

**Issue by CTFishUSA** [bitsy-ai/octoprint-nanny-plugin#95]
- OctoPrint version: Website https://print-nanny.com/dashboard/octoprint-devices/
- Print Nanny version: n/a
- OctoPi (or other OS) version: n/a

**Describe the bug**
Rightmost column is off the page.  Like the frame is not scaling properly for my screen.  If this is a code issue on your end, suggest to fix.  If on my end (Raspberry 4 with Chromium), close with no change.

**To Reproduce**
1. Visit https://print-nanny.com/dashboard/octoprint-devices/
2. See error

**Expected behavior**
The page should fit all of my information (RAM is cut off).  Is it because my OctoPrint Device name is too long?  Can you add a scrollbar?

![2021-01-26-205241_1680x1050_scrot](https://user-images.githubusercontent.com/76830938/105931048-81e9d900-6018-11eb-96c2-30da1e0678b1.png)

**Screenshots**

**Comment by leigh-johnson** [comment]
Thank you! I'll add some overflow or word wrap

![a9919cebd23ce8f21d74c974ac919c2f](https://user-images.githubusercontent.com/2601819/106103468-d9aa4200-60f5-11eb-9ca6-4379d915649f.jpg)

**Comment by leigh-johnson** [comment]
I ended up sticking most of these fields behind a "view metadata" button. You'll see this update in release 0.4.2, which I'll roll out in an hour or three. Thanks again for reporting this! Closing this issue for now. 

![Screenshot from 2021-01-31 12-44-59](https://user-images.githubusercontent.com/2601819/106397424-2fb30a00-63c2-11eb-9d46-568fcc6e4d5f.png)

Actual footage of me adjusting the CSS and then giving up

https://media1.tenor.com/images/614c9b4639a2588383f47e138177da81/tenor.gif?itemid=12014506
